doc/api/esm.md

```diff
@@ -5,6 +5,10 @@
 <!-- YAML
 added: v8.5.0
 changes:
+  - version:
+    - REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/37468
+    description: Consolidate loader hooks.
```

Comment:
It might be worth explaining in more details what changed.
```suggestion
    description: Consolidate loader hooks: removed `getFormat`, `getSource`,
                 `transformSource`, and `getGlobalPreloadCode` hooks and added
                 `load` and `globalPreload` hooks.
```